esmvaltool/cmorizers/data/formatters/datasets/cds_satellite_lai_fapar.py

```diff
@@ -170,7 +171,7 @@ def _set_time_bnds(in_dir, var):
     return cubelist
 
 
-def cmorization(in_dir, out_dir, cfg, cfg_user):
+def cmorization(in_dir, out_dir, cfg, cfg_user, start, end):
```

Comment:
Script works fine but I'm not sure what the arguments `start` and `end` are used for.

Reply:
For nothing here, some others may make use of it. Ideally, in the future all datasets that can be downloaded by periods should support this to allow users to download and cmorize only the years that are missing, being those the rest of the dataset after testing the procedure with a single year or the latest years of an ongoing reanalysis or observation product